pushy-donkeys/Bangazon-Site
# my account edit profile order history maybe

Fixes for #17 #16 

Changes proposed in this pull request:
- Added a method in ManageController for getting the routes for CurrentUser
- Added a view model for editing your profile
- Modified Index in Manage to give us correct links 
- Links include 'Add Payment option', 'Edit Profile', 'Order History', 'Saved Payment Methods'
- added 'Editprofile" in Manage 
- Added a method in OrderController for getting current users order history
- Added a View model for OrderProduct to get the past order
-Added a view in Order called OrderHist that is a list of past orders

How to test:
- Pull down branch / run program
- go to "My Profile" / choose an option
- go through each of the links and test them


*NOTE* I DO NOT KNOW FOR CERTAIN IF THE ORDER HISTORY WORKS SINCE WE DON'T HAVE TICKET 2 WORKING ATM, BUT IT SHOULD PROBABLY MAYBE KINDA SORTA WORK.

@PushyDonkey/
